packages/tables/resources/views/columns/select-column.blade.php

```diff
@@ -48,7 +48,7 @@
 >
     <input
         type="hidden"
-        value="{{ str($state)->replace('"', '\\"') }}"
+        value="{{ $state }}"
```

Comment:
```suggestion
        value="{{ str(strval($state))->replace('"', '\\"') }}"
```

Reply:
How does this perform?

Reply:
Tried that out, did not work unfortunately. 

I had to add a bit more logic to the on change to satisfy the following:

`select-column.blade.php`
```php
const response = await $wire.updateTableColumnState(
```

which ultimately calls `CanUpdateState`
```php
$record->setAttribute($columnName, $state);
```

which fails in Laravel core from:

```php
if ($this->isEnumCastable($key)) {
    $this->setEnumCastableAttribute($key, $value);

    return $this;
}
```

because we were setting `""` to the enum value and it was trying to cast that to the backed enum. 
Instead we'll have to send `null` as a value.

Before:
```php
$record->setAttribute($columnName, $state /* "" */);
```

After:
```php
$record->setAttribute($columnName, $state /* null */);
```

Which tries to take the empty string and cast it to the backed enum, throwing an exception.

Let me know what you think! This fixes it for me :)